packages/core/src/index.ts

```diff
@@ -148,34 +145,20 @@ class Hopr extends EventEmitter {
    * @param options
    * @param provider
    */
-  public constructor(private id: PeerId, private options: HoprOptions) {
+  public constructor(
+    private id: PeerId,
+    private db: HoprDB,
+    private chain: HoprCoreEthereum,
+    private options: HoprOptions,
+    private publicNodesEmitter: HoprConnectOptions['publicNodes'] = new EventEmitter()
+  ) {
     super()
 
     if (!id.privKey || !isSecp256k1PeerId(id)) {
       throw new Error('Hopr Node must be initialized with an id with a secp256k1 private key')
     }
-    this.db = new HoprDB(
-      PublicKey.fromPrivKey(id.privKey.marshal()),
-      options.createDbIfNotExist,
-      VERSION,
-      options.dbPath,
-      options.forceCreateDB
-    )
     this.environment = options.environment
-
-    const provider = expandVars(this.environment.network.default_provider, process.env)
     log(`using environment: ${this.environment.id}`)
-    log(`using provider URL: ${provider}`)
```

Comment:
Please add an equivalent print out somewhere, this is useful during development and testing.

Reply:
Restored the logging during `createHoprNode`